client/post-editor/post-editor.jsx

```diff
@@ -189,10 +189,19 @@ export const PostEditor = React.createClass( {
 		return this.props.setLayoutFocus( 'content' );
 	},
 
-	setConfirmationSidebar: function( state ) {
+	setConfirmationSidebar: function( { state, context = null } ) {
```

Comment:
Just for historical sake - I mentioned this in Slack, but the "state" argument could be a little misleading given React's state, but I don't know that it's a blocker.